scene/gui/rich_text_label.cpp

```diff
@@ -256,6 +256,10 @@ int RichTextLabel::_process_line(ItemFrame *p_frame, const Vector2 &p_ofs, int &
 			lh = line < l.height_caches.size() ? l.height_caches[line] : 1;                                                                                     \
 			line_ascent = line < l.ascent_caches.size() ? l.ascent_caches[line] : 1;                                                                            \
 			line_descent = line < l.descent_caches.size() ? l.descent_caches[line] : 1;                                                                         \
+			if (p_mode == PROCESS_DRAW) {																														\
+				if (line < l.offset_caches.size()) 																											    \
+					wofs = l.offset_caches[line];																												\
+			}																																					\
```

Comment:
Thanks for fixing this! I don't know about the logic in this part of code, but I just want to point out that `\` characters are not properly aligned.
I'm going to run some tests on my side for the fix itself.

Reply:
And brackets are also missing for the second condition to follow the coding rules :)

Reply:
I've added the brackets with the second condition. But still can't properly align the '\'. In my IDE, they're properly aligned, but here they're not aligned. Don't know what to do

Reply:
Your IDE probably uses more or less than 4 spaces for tabs.
You can fix alignment using `clang-format`: https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes

Reply:
I'm getting this error while formatting in visual studio 17 : Unknown key 'JavaImportGroups'

Reply:
That means that your clang-format version is probably too old. You need 8.0 or later.